### ISSUE TITLE
G18REF   Aml8726MX upgrade to Openelec 5.0.3

### ISSUE THREAD

**Issue by Kezzi** [codesnake/OpenELEC.tv#52]
My MX is always locking up at the Openelec Screen after I try to update to 5.0.3
I tried a new install and I get the same result the screen goes black then I restart and the openelec screen comes on with Fuzzy retrace lines all over it them nothing more!
Any help would be much appreciated Thank you!!!

**Comment by codesnake** [comment]
It seems this problem is present only on some of the MX2 boxes, at least I cannot reproduce it on my MX2.

**Comment by Kezzi** [comment]
Thank You for the prompt reply! Does this means I should buy another box? Or wait for 5.0.5?
In any case your work is much appreciated and Many Thanks for your Efforts!!!

**Comment by Diezl** [comment]
try going back to 1.15 then up 1.16 then to open elec seems some old trces get stuck in the mem of the box make sure its the right firmware for 5.03 i just loaded m8 on my mx2 and got brick dont buy a new box yet gl

**Comment by Kezzi** [comment]
OK Thanx I will try it!
PS: where can I find these files?

**Comment by Diezl** [comment]
i uplouded them to my drop box theres a tutorial remember this will take you to matricom 115 then matricom 116 then flash the openelec zip its the release from codesnake here 

https://www.dropbox.com/sh/hqxv0hrbqgmv4q0/AADuMQSSjvBRVfOLp6xO4plHa?dl=0
good luck check your box for heat

**Comment by Kezzi** [comment]
God it has been a long try for this but here goes!
I tried the Matricon Aug9 file then Openelec No go Black screen
Next I tried Matricon 1.15 and followed up with 1.16 then openelec 5.0.3 No go same problems
The only file I had any success with was Matricon G18ref_MX2_4.7.1 it works!!!
Sad story anyway Thank You Very Very Much for your Help!!!!
You Tried and that is what counts the most!!!

**Comment by Diezl** [comment]
so your box is running Ya?

**Comment by Diezl** [comment]
im locked in black screen my next step is UART lol cant waste a box

**Comment by Kezzi** [comment]
Well it runs but not great especially with Live Streams! It locks up like a stuck CD playing music!
Best of luck with yours!

**Comment by Kezzi** [comment]
One more time here I go!
I found this file it seems to be by a Russian coder because it has a lot of Russian in it but it it works very
well the best so fat ti is stable and it is also Android but seems to lack Google Tracking so no Buffering!
I had to uninstall the XBMC on it and install Kodi 14.1 Now it is running perfect perhaps a solution for everyone else with the Chinese version of MX! I found it ob a link on Youtube!
http://matriserver.com/files/MX2/firm...
